onnx_caffe2/backend.py

```diff
@@ -164,7 +165,7 @@ class Caffe2Backend(Backend):
     # so this interface MAY make BC-breaking changes.  Specify an
     # opset_version if you don't want this to version.
     @classmethod
-    def run_node(cls, node, inputs, device='CPU', opset_version=_known_opset_version):
+    def run_node(cls, model_helper, node, inputs, device='CPU', opset_version=_known_opset_version):
```

Comment:
this is not what you want. This is an external API to run a single node. I suppose you want to change RunOperatorsOnce below to RunNetOnce(init_net), RunNetOnce(predict_net)